src/config.js

```diff
@@ -0,0 +1,6 @@
+const API_URLS = {
+  SIGNUP: 'http://10.58.0.184:8000/user/signup',
+  SIGNIN: 'http://10.58.0.184:8000/user/signin',
+};
```

Comment:
👍 하지만 요렇게 수정하면 좀 더 base url 자체가 수정되었을때 쉽게 대응할 수 있어 보이네요
```suggestion
const BASE_URL = "http://10.58.0.184:8000"
const API_URLS = {
  SIGNUP: `${BASE_URL}/user/signup`,
  SIGNIN: `${BASE_URL}/user/signin`,
};
```